tasks/main.yml

```diff
@@ -1,28 +1,33 @@
+# SPDX-License-Identifier: MIT
 ---
-# tasks file for template
+# Put the tasks for your role here.
 
+# An example of how to set distribution and version specific internal variables
+# (defined in vars/ directory):
 - name: Set version specific variables
   include_vars: "{{ item }}"
   with_first_found:
+    - "{{ ansible_distribution }}-{{ ansible_distribution_version }}.yml"
     - "{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
     - "{{ ansible_distribution }}.yml"
     - "{{ ansible_os_family }}.yml"
     - "default.yml"
 
-- name: Install packages
-  package:
-    name: "{{ SOME_PACKAGE_NAMES_VARIABLE | default([]) }}"
-    state: present
-
-- name: Enable SUBSYSTEM service
-  service:
-    name: "{{ SUBSYSTEM_service_name }}"
-    state: started
-    enabled: yes
-
-- name: Generate /etc/SUBSYSTEM.conf
-  template:
-    src: SUBSYSTEM.conf.j2
-    dest: /etc/SUBSYSTEM.conf
-    backup: yes
-  notify: restart SUBSYSTEM
+# Examples of some tasks:
+# - name: Install packages
+#   package:
+#     name: "{{ __template_packages }}"
+#     state: present
+#
+# - name: Enable {{ __template_service }} service
+#   service:
+#     name: "{{ __template_service }}"
+#     state: started
+#     enabled: yes
+#
+# - name: Generate /etc/{{ __template_config }}
+#   template:
+#     src: "{{ __template_config }}.j2"
+#     dest: /etc/{{ __template_config }}
+#     backup: yes
+#   notify: template handler to restart service
```

Comment:
This task could be left uncommented to let the tests actually do anything. (Ditto for package task, if the list of packages is left empty.)